test/e2e/volume_provisioning.go

```diff
@@ -160,7 +174,7 @@ func newClaim(ns string, alpha bool) *v1.PersistentVolumeClaim {
 
 	if alpha {
 		claim.Annotations = map[string]string{
-			storageutil.AlphaStorageClassAnnotation: "",
+			"volume.alpha.kubernetes.io/storage-class": "anything",
```

Comment:
would it make sense to not have an annotation in order to test the "default" behavior?